### PR TITLE
Remove redundant alias import and replace few function names in ISCSI server

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

```release-note-none

```

### CODE REVIEW

**Review by ddebroy — COMMENTED** [review]
One suggestion. Otherwise LGTM.

File: internal/server/disk/internal/v1beta1/conversion.go [hunk 1]
```diff
@@ -3,8 +3,8 @@ package v1beta1
 // Add manual conversion functions here to override automatic conversion functions
 
 import (
-	v1beta1 "github.com/kubernetes-csi/csi-proxy/client/api/disk/v1beta1"
```
[the remark this answers is not in the document]
Comment (humblec) [reply internal/server/disk/internal/v1beta1/conversion.go:6]: Sure.. Updated patch set. PTAL. Thanks @ddebroy

**Review by ddebroy — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**